### DISCUSSION TITLE
en ছাড়া অন্য কোনও লেআউট থেকে bn এ সুইচ করলে বাংলায় ইনপুট নেয়না।

### DISCUSSION THREAD

**Discussion by rashidi77** [OpenBangla/OpenBangla-Keyboard#265] [General]
সমস্যা : ibus এর লেআউট যদি en (ইংরেজি) ছাড়া অন্য কোনও লেআউট থেকে সরাসরি bn-এ সুইচ করি তখন বাংলায় ইনপুট না নিয়ে আগের লেআউটেই ইনপুট নেয়। যেমন ar (আরবি) থেকে লেআউট সুইচ করে bn করলে বাংলার পরিবর্তে আরবিতে ইনপুট নেয়।
[উল্লেখ্য : জাপানিজ Anthy কীবোর্ডেও এই সমস্যাটা আছে।]

সাময়িক সমাধান : ar থেকে সরাসরি bn এ সুইচ না করে প্রথমে en এ সুইচ করি, তারপর bn এ সুইচ করি। এভাবে ar>en>bn । দুইবার করে windows+space চাপতে হয়। এতে দ্রুত লেখার ব্যাঘাত ঘটে। কিন্তু অন্য কোনও সমাধান পাইনি।

উদাহরণত : 
https://user-images.githubusercontent.com/93915281/143670143-76ea07a5-a61a-4e97-8129-2c9842838f1b.mp4

OBK version : 2.00 & 3.00
Input Method : ibus (not tested on fcitx5)

**Comment by RezwanArefin01** [comment]
Any updates on this issue?

**Comment by RezwanArefin01** [comment]
Tested on fcitx5. Works as expected. :tada:
